cmd/maestro/servecmd/cmd.go

```diff
@@ -78,6 +80,16 @@ func runServer(cmd *cobra.Command, args []string) {
 
 	ctx, cancel := context.WithCancel(context.Background())
 
+	tracingShutdown := func(context.Context) error { return nil }
+	log := logger.NewOCMLogger(ctx)
```

Comment:
why create a new logger? Also `logger.NewOCMLogger()` doesn't seem to be used anywhere else in the project :thinking:

Reply:
It is there. I can switch it over the zap logger which is used in this repo. I was planning to switch over when I import the ocm library in the future but I can do it now.

Reply:
Done.